docs/collecting-data/collecting-from-own-applications/react-native-tracker/anonymous-tracking/index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Anonymous Tracking"
+date: "2022-08-30"
+sidebar_position: 25
+---
+
+# Anonymous Tracking
+
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+```
+
+:::info
+
+This feature is available since v1.3.
+
+:::
+
+Anonymous tracking is a tracker feature that enables anonymising various user and session identifiers to support user privacy in case consent for tracking the identifiers is not given.
+
+The affected user and session identifiers are stored in two context entities: [Session](../tracking-events/index.md#session) and [Platform context](../tracking-events/index.md#platform). The Session context entity contains user and session identifiers, while the Platform context entity contains user identifiers. Concretely, the following user and session identifiers can be anonymised:
+
+1. Client-side user identifiers: the `userId` in Session context entity and the IDFA identifiers (`appleIdfa`, and `appleIdfv`) in the Platform context entity.
+2. Client-side session identifiers: `sessionId` in Session context.
+3. Server-side user identifiers: `network_userid` and `user_ipaddress` event properties.
+
+There are several levels to the anonymisation depending on which of the three categories are affected:
+
+## 1. Full client-side anonymisation
+
+In this case, we want to anonymise both the client-side user identifiers as well as the client-side session identifiers. This means disabling the Session context altogether and enabling user anonymisation:
+
+```typescript
+const tracker = createTracker(
+    'appTracker',
+    {endpoint: COLLECTOR_URL},
+    {
+        trackerConfig: {
+            sessionContext: false, // Session context entity won't be added to events
+            userAnonymisation: true // User identifiers in Platform context (IDFA and IDFV) will be anonymised
+        }
+    }
+);
+```
+
+## 2. Client-side anonymisation with session tracking
+
+This setting disables client-side user identifiers are but tracks session information. In practice, this means that events track the Session context entity but the `userId` property is a null UUID (`00000000-0000-0000-0000-000000000000`). In case Platform context is enabled, the IDFA identifiers will not be present.
```

Comment:
```suggestion
This setting disables client-side user identifiers but tracks session information. In practice, this means that events track the Session context entity but the `userId` property is a null UUID (`00000000-0000-0000-0000-000000000000`). In case Platform context is enabled, the IDFA identifiers will not be present.
```